src/App.java

```diff
@@ -12,20 +12,209 @@ public void start() {
         String userName = getUserName();
         printGreeting(userName);
         printInstructions();
-        doMadlib();
-        System.out.println("hi");
+        String choice = choiceMadLibs();
+       if (choice.equals("1")) {
+        doMadlibIntro1();
+        doMadlib1();
+       } else{
+        doMadlibIntro2();
+        doMadlib2();
+       }
+        System.out.println();
     }
     
     /* Here you should create all missing methods used by the start() method
      * I have created the signature for ONE of the methods below.
      * You can create any other methods you need as well!
      */ 
     
+     public void printGreeting(String name){
```

Comment:
You did not include the documentation for this method or any other, please be careful with that, it was listed as required in the Readme